tests/quarkus/pom.xml

```diff
@@ -38,15 +34,13 @@
       <dependency>
         <groupId>io.hawt</groupId>
         <artifactId>hawtio-bom</artifactId>
-        <version>${project.version}</version>
+        <version>${hawtio.bom.version}</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
     </dependencies>
   </dependencyManagement>
-
   <dependencies>
-    <!-- Camel Quarkus -->
```

Comment:
Is there a reason you remove these comments?  I guess you ran the quarkus command which automatically removes them. Maybe we might want to restore them.